deploy/crds/hco00.crd.yaml

```diff
@@ -117,14 +117,14 @@ spec:
                 type: object
               featureGates:
                 default:
-                  sriovLiveMigration: false
+                  sriovLiveMigration: true
                   withHostPassthroughCPU: false
                 description: featureGates is a map of feature gate flags. Setting
                   a flag to `true` will enable the feature. Setting `false` or removing
                   the feature gate, disables the feature.
                 properties:
                   sriovLiveMigration:
-                    default: false
+                    default: true
                     description: Allow migrating a virtual machine with SRIOV interfaces.
                       When enabled virt-launcher pods of virtual machines with SRIOV
```

Comment:
Please remove this part _When enabled virt-launcher pods of virtual machines with SRIOV
interfaces run with CAP_SYS_RESOURCE capability. This may degrade
virt-launcher security._

As we did on #1375